Sources/Fluent/Model.swift

```diff
@@ -0,0 +1,101 @@
+
+public protocol Model {
+    static var entity: String { get }
+    var id: String? { get }
+    
+    func serialize() -> [String: StatementValueType]
```

Comment:
Could we call `StatementValueType` just `Value`? `Type` is unnecessary as of new Swift API Guidelines, statement also seems redundant.